src/profile_manager/views.py

```diff
@@ -157,6 +229,35 @@ def get_object(self):
         return self.request.user.profile
 
 
+@method_decorator(staff_member_required, name='dispatch')
+class PasswordReset(FormView):
+    form_class = SetPasswordForm
+    template_name = 'profile_manager/password_change_form.html'
+    
+    def get_instance(self):
+        model_pk = self.kwargs['pk']
+        return get_user_model().objects.get(pk=model_pk)
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        profile = self.get_instance().profile
+
+        context['heading'] = "Changing " + profile.user.get_username() + "'s Password"
+        context['submit_btn_value'] = "Update"
+        
+        return context
+
+    def get_form(self):
+        return PasswordReset.form_class(user=self.get_instance(), **self.get_form_kwargs())
+
+    def form_valid(self, form):
+        form.save()
```

Comment:
Is this necessary?  Doesn't the form save already with default behavior?  Not sure why you're overriding this method

Reply:
necessary since it doesnt save without form.save()

also all of the inherited classes dont even save the form. But there are definitely classes i missed since its hard to find all the inherited classes

also manually tested this by removing form.save(), form doesnt save